routes/api.js

```diff
@@ -41,6 +41,30 @@ router.all('/*', (req, res, next) => {
 
 });
 
+router.post('/listadd', (req, res) => {
```

Comment:
Hello @antoninchadima, good implementation I used she in a project in devlopment. But I think is more expressive use in call of API `/lists/add` as `/blacklist` URLs

Reply:
Is this the only pending change to have this useful feature available merged?